src/renderer/domains/collectives/model/referendum/model.ts

```diff
@@ -26,19 +26,19 @@ const { pending, subscribe, unsubscribe, received, fulfilled } = createDataSubsc
 >({
   initial: $list,
   fn: ({ api, palletType }, callback) => {
-    let currectAbortController = new AbortController();
+    let currentAbortController = new AbortController();
```

Comment:
This name seems a bit cleaner
```suggestion
    let abortController = new AbortController();
```